translations/en.json

```diff
@@ -125,7 +125,7 @@
     "bug-report-submit-email": "Submit by email with logs archive",
     "bug-report-submit-gh-issue": "Submit a GitHub issue without logs",
     "bug-report-too-short-description": "Description is too short",
-    "by-continuing-you-accept": "By continuing you accept our ",
+    "by-continuing-you-accept": "By continuing you accept\nour ",
```

Comment:
could you elaborate?

Reply:
This is to effectively push the text `our` to next line. It was required in the ticket.

Reply:
I think it's better to handle it with style(using `width`/`max-width`) instead of updating the translation text. @flexsurfer wdyt?

Reply:
it depends, but I agree it looks strange, will it work only for English? what about different screen sizes? so if we always want it on the new line no matter what by design then probably its ok, but im not sure

Reply:
@ajayesivan, @flexsurfer yes, it looks strange but requirement was to split from that particular word(otherwise previous design as well could’ve put ). I did check the translation part as well. Adding a hardcoded max-width did seem right as well so I added line break. I'll use max-width now..